### DISCUSSION TITLE
How to add my own icons?

### DISCUSSION THREAD

**Discussion by ajfriesen** [google/docsy#1340] [Q&A]
I would like to use an icon that is not available in font awesome on the home page `_index.html`

How would I do that with this icon for example?

https://materialdesignicons.com/icon/home-assistant

**Comment by AntoineGlacet** [comment]
ever had an answer for that?

**Comment by fekete-robert** [comment]
Hi, it depends where you'd like to use the icon:
- if you just want to use the icon in the regular content, then you can download the linked icons as svg, put it into the static directory, and use it as a regular html image <img src="/your-icon.svg">
- if you want to replace the fontawesome icon in one of the block shortcodes (like `{{% blocks/feature icon="fa-lightbulb"`) then you have to modify and customize the shortcode, because that's currently designed to use fontawesome icons.
